modules/DNN/RBM.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "types.h"
+#include "NeuronLayer.h"
+
+namespace DirectGraphicalModels {
+	namespace dnn {
+		class CRBM {
+			public:
+				DllExport CRBM(const std::vector<ptr_nl_t>& vpLayers);
+				DllExport CRBM(const CRBM&) = delete;
+				DllExport ~CRBM(void) = default;
+
+				DllExport bool operator=(const CRBM&) = delete;
+
+				DllExport void debug(); //printing out rows and cols just to be sure they are correct
+
+				DllExport Mat getBinomial(Mat mean);
```

Comment:
- All the matrices is better to pass not by value, but by reference
- Please add some comments to the methods and member-variable. What do they do, why are they needed, _etc._